apps/backend/scripts/seed_e2e_data.py

```
@@ -174,15 +173,15 @@ def verify_seed(db):
     else:
-        print(f"   ❌ User not found!")
+        print("   ❌ User not found!")
         return False
 
     # Verify dataset
     dataset = db['user_data'].find_one({'user_id': str(user['_id'])})
     if dataset:
         print(f"   ✅ Dataset verified: {dataset['filename']}")
     else:
-        print(f"   ℹ️  No datasets found (this is OK if not needed)")
+        print("   ℹ️  No datasets found (this is OK if not needed)")
```

Ruff (0.14.10)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)